### ISSUE TITLE
Some builds don't have an 'extra' value

### ISSUE THREAD

**Issue by zmc** [ceph/shaman#61]
Example:

https://shaman.ceph.com/api/search/?status=ready&project=ceph&flavor=default&distros=centos%2F7%2Fx86_64&sha1=0492c232b2463b81f48f7974c7898b2e9a364743
```
[
{
status: "ready",
sha1: "0492c232b2463b81f48f7974c7898b2e9a364743",
extra: { },
url: "https://1.chacra.ceph.com/r/ceph/master/0492c232b2463b81f48f7974c7898b2e9a364743/centos/7/flavors/default/",
distro_codename: null,
modified: "2016-09-14 21:23:46.483273",
distro_version: "7",
project: "ceph",
flavor: "default",
ref: "master",
chacra_url: "https://1.chacra.ceph.com/repos/ceph/master/0492c232b2463b81f48f7974c7898b2e9a364743/centos/7/flavors/default/",
archs: [
"x86_64",
"source"
],
distro: "centos"
}
]
```

**Comment by andrewschoen** [comment]
That repo was most likely created before we added in the ``extra`` functionality.

Once we've got things somewhat stable we're gonna need to wipe all this test and incomplete data from the systems.

**Comment by andrewschoen** [comment]
@zmc the modified date is like that because I forced updated all the repos in chacra1 yesterday. That modified date is when the repo object in shaman has been updated.

The ``extra`` data comes from the actual build, there is no way to populate that on these old repos unless we rebuild the binaries.

It looks like @alfredodeza must have done some cleanup on chacra1 last night, I just forced an update on chacra2 right now as I was seeing some missing repos at https://shaman.ceph.com/api/search/ for chacra2.
